app/Http/Controllers/StudentCouncil/EconomicController.php

```diff
@@ -107,8 +106,8 @@ public function payKKTNetreg(Request $request)
         $valasztmany_checkout = Checkout::studentsCouncil();
         $admin_checkout = Checkout::admin();
 
-        $this->authorize('handle', $valasztmany_checkout);
-        $this->authorize('handle', $admin_checkout);
+        $this->authorize('addPayment', $valasztmany_checkout);
+        $this->authorize('addPayment', $admin_checkout);
```

Comment:
Altough economic-members add these transactions to the admin checkout, these users may not have access to the admin checkout.

Reply:
I would say working as intended. The economic committee could collect the payments, but not see more details. This could be changed, when more functionality is added.
Currently I found the admin and students' council checkouts a bit too much mixed together. Once we have clean, separate functionality for the two, we could modify the permissions.

Reply:
This would mean that all economic committee member who receives kkt/netreg need addPayment authorization for both admin's and student_committee's checckout.

Reply:
This is the only thing that the two checkout mixes together because the way we collect netreg through economic committee members.